packages/ses/package.json

```
@@ -42,6 +42,10 @@
       "import": "./index.js",
       "require": "./dist/ses.cjs"
     },
+    "./util": {
+      "import": "./dist/util.js",
+      "require": "./dist/util.cjs"
+    },
```

I don’t think we have to bundle the utils, but because some tools don’t recognize the `"exports"` field, we do need the key and value to match. We also have an emerging precedent for a file like this having the name `./tools.js` (`@endo/pass-style` will have one soon if it hasn’t landed yet). So my overall recommendation is for this to be:

```suggestion
    "./tools.js": "./tools.js",
```

And just to rename `utils.js` to `tools.js`.